spec/lib/tasks/import_spec.rb

```diff
@@ -13,5 +14,14 @@
     it "creates a document" do
       expect { Rake::Task["import:whitehall"].invoke("123") }.to change { Document.count }.by(1)
     end
+
+    it "aborts if the import fails" do
+      expect(WhitehallImporter::Import).to receive(:call).and_raise("Error importing")
+
+      expect($stdout).to receive(:puts).with("Import failed")
+      expect($stdout).to receive(:puts).with("Error: Error importing")
+      expect { response = Rake::Task["import:whitehall"].invoke("123") }
```

Comment:
```suggestion
      expect { Rake::Task["import:whitehall"].invoke("123") }
```

Linter is complaining about useless variable.

Reply:
D'oh thanks!